common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -781,6 +781,24 @@ colored_emblem_texture_lists = {
 			}
 			100 	= "ce_kirintor_eye.dds"
 		}
+		
+		### Death
+		special_selection = {
+			trigger = {
+				death_coa_trigger = yes
+			}
+			100 	= "ce_abomination.dds"
+			100 	= "ce_dk_helmet.dds"
+			100 	= "ce_necropolis.dds"
+			100 	= "ce_ram_skull.dds"
+			100 	= "ce_scream.dds"
+			100 	= "ce_skull1.dds"
+			100 	= "ce_skull1_fire.dds"
+			100 	= "ce_skull2.dds"
+			100 	= "ce_skull3.dds"
```

Comment:
Should be bigger and lowered.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126882281-f43a3374-4c4e-4484-96fa-f834d1c4ffed.png)
![image](https://user-images.githubusercontent.com/46384992/126882308-bb4bd5af-4f92-4cda-99a1-cbe621069e10.png)

</details>

Reply:
Still too small.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126907216-acc91f04-973b-455f-ad0f-52079de2c74d.png)

</details>